tests/ControllerTest.php

```diff
@@ -210,6 +211,22 @@ public function testUseCaseDatabaseOnlyDefinedOnOneEnd(): void
         $this->assertSame($lines, explode("\n", $output->fetch()));
     }
 
+    public function testPhpApiReadmeExample()
```

Comment:
since we commented this out in the readme, i dont think we should for now have this in the tests either

Reply:
A test more dosen't hurt.